rotkehlchen/tests/unit/decoders/test_stakedao.py

```diff
@@ -304,3 +314,115 @@ def test_withdraw(ethereum_inquirer, ethereum_accounts, stakedao_gauges):
+        notes=f'Deposit {deposit_amount} 2BTC-ng in StakeDAO',
+        counterparty=CPT_STAKEDAO,
+        address=string_to_evm_address('0xfc072ead21DdA7260008aa4165907Bc27cC59329'),
+    ), EvmEvent(
+        tx_hash=evmhash,
+        sequence_index=15,
+        timestamp=timestamp,
+        location=Location.ARBITRUM_ONE,
+        event_type=HistoryEventType.RECEIVE,
+        event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+        asset=Asset('eip155:42161/erc20:0x0B31dA9ff8b35106B86B6203cebf94603131c75c'),
+        amount=FVal(deposit_amount),
+        location_label=user_address,
+        notes=f'Receive {deposit_amount} sd2BTC-ng-vault-gauge after depositing in StakeDAO',
+        counterparty=CPT_STAKEDAO,
+        address=ZERO_ADDRESS,
+    )]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('binance_sc_accounts', [['0xa99345367044C80D8f01d0618c44B752C4C29Bdb']])
+def test_withdraw_bsc(binance_sc_inquirer, binance_sc_accounts, stakedao_gauges):
+    tx_hex = deserialize_evm_tx_hash('0x0e8556a645758a6747072673fa713f98a5b3dae090b940257e5ef7c86dc73d49')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = binance_sc_accounts[0]
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=binance_sc_inquirer, tx_hash=tx_hex)
+    gas_amount, cake_reward, received_amount, timestamp = '0.000358695', '1.156230303623121164', '2000', TimestampMS(1743674447000)  # noqa: E501
+    assert events == [EvmEvent(
+        tx_hash=evmhash,
+        sequence_index=0,
+        timestamp=timestamp,
+        location=Location.BINANCE_SC,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        asset=A_BSC_BNB,
+        amount=FVal(gas_amount),
+        location_label=user_address,
+        notes=f'Burn {gas_amount} BNB for gas',
+        counterparty=CPT_GAS,
+        address=None,
+    ), EvmEvent(
+        tx_hash=evmhash,
+        sequence_index=1,
+        timestamp=timestamp,
+        location=Location.BINANCE_SC,
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+        asset=Asset('eip155:56/erc20:0xE36c375AD28C0822ab476Bd62A8BEAd88d9a4e34'),
+        amount=FVal(received_amount),
+        location_label=user_address,
+        notes=f'Return {received_amount} PCS-ERC20-gauge to StakeDAO',
+        counterparty=CPT_STAKEDAO,
+        address=string_to_evm_address('0xE36c375AD28C0822ab476Bd62A8BEAd88d9a4e34'),
+    ), EvmEvent(
+        tx_hash=evmhash,
+        sequence_index=2,
+        timestamp=timestamp,
+        location=Location.BINANCE_SC,
+        event_type=HistoryEventType.RECEIVE,
+        event_subtype=HistoryEventSubType.REWARD,
+        asset=Asset('eip155:56/erc20:0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82'),
+        amount=FVal(cake_reward),
+        location_label=user_address,
+        notes=f'Claim {cake_reward} Cake from StakeDAO',
```

Comment:
hmm this looks wrong. The claim should not be between the return/withdraw. It should come either after or before.